cli/cmd/sync_v3.go

```diff
@@ -197,8 +197,8 @@ func syncConnectionV3(ctx context.Context, syncOptions syncV3Options) (syncErr e
 		}
 	}
 
-	log.Info().Str("source", sourceSpec.VersionString()).Strs("destinations", destinationStrings).Strs("transformers", transformerStrings).Time("sync_time", syncTime).Msg("Start sync")
-	defer log.Info().Str("source", sourceSpec.VersionString()).Strs("destinations", destinationStrings).Strs("transformers", transformerStrings).Time("sync_time", syncTime).Msg("End sync")
+	log.Info().Str("source", sourceSpec.VersionString()).Str("source_name", sourceName).Str("invocation_id", invocationUUID.UUID.String()).Strs("destinations", destinationStrings).Strs("transformers", transformerStrings).Time("sync_time", syncTime).Msg("Start sync")
```

Comment:
Invocation Id is already added in https://github.com/cloudquery/cloudquery/blob/fed76694d622e511f7d61c2d0baedfbe39b15d60/cli/cmd/logging.go#L51 I think

Reply:
We could use the same approach to add the source/destination name to each plugin logger in https://github.com/cloudquery/cloudquery/blob/fed76694d622e511f7d61c2d0baedfbe39b15d60/cli/cmd/sync.go#L217 and https://github.com/cloudquery/cloudquery/blob/fed76694d622e511f7d61c2d0baedfbe39b15d60/cli/cmd/sync.go#L266 while we're at it